amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -1062,6 +1102,19 @@ void ABLBoundaryPlane::read_file(const bool nph_target_time)
     const amrex::Real time =
         nph_target_time ? m_time.current_time() + 0.5 * m_time.delta_t()
                         : m_time.new_time();
+
+#ifdef ERF_AMR_WIND_MULTIBLOCK
+    if (m_out_fmt == "erf-multiblock") {
+        ReadERFFunction read_erf = *m_read_erf;
+        if (read_erf) {
```

Comment:
the linters like to be explicit:
```suggestion
        if (read_erf != nullptr) {
```